src/technologies/g.json

```diff
@@ -450,19 +450,11 @@
     "description": "Premium block library & block editor enhancements from StudioPress.",
     "icon": "genesis-blocks.png",
     "requires": "WordPress",
+    "dom": [
+      "link[href*='/wp-content/plugins/atomic-blocks']", 
+      "link[href*='/wp-content/plugins/genesis-blocks']"
+    ],
     "html": "<div[^>]+class=[\"']wp-block-atomic-blocks-*",
-    "dom": {
-      "link[href*='/wp-content/plugins/atomic-blocks']": {
-        "attributes": {
-          "href": "/wp-content/plugins/atomic-blocks/"
```

Comment:
I thought you said on the last PR that these were needed? But now they're not? Confused...

Reply:
Been testing more, the markup I use now works. Turns out if you wanted to strip the `attributes` bit you either have to set `"required": true` or do as I do now.

Reply:
OK as long as there's a reason!